### PR TITLE
Update to support Tactician

### PR DESCRIPTION
**Pull request by ameech**
Updated to be a simple queue implementation for Tactician.

@equip/contributors please take a look!

### CODE REVIEW

File: src/QueueMiddleware.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Equip\Queue;
+
+use League\Tactician\Middleware;
+
+class QueueMiddleware implements Middleware
```
Comment (ameech) [inline src/QueueMiddleware.php:8]: This class needs tests yet.

**Comment by shadowhand** [comment]
:+1:

File: src/QueuedCommand.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Equip\Queue;
+
+final class QueuedCommand
```
Comment (ameech) [inline src/QueuedCommand.php:5]: This class is required so we don't keep queuing up the same command over and over.

File: src/QueueMiddleware.php [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+     *
+     * @param string $command_name
+     *
+     * @return string
+     *
+     * @throws \Exception If the command isn't assigned to a pipe
+     */
+    private function getQueue($command_name)
+    {
+        // TODO: refactor this?
```
Comment (fentie) [inline src/QueueMiddleware.php]: `array_search`?

**Pull request merged**